RA_Integration.sln

```diff
@@ -12,35 +12,72 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "RA_Integration.Tests", "tests\RA_Integration.Tests.vcxproj", "{876DEBDF-C7C8-4713-8CEF-EBDEE65306A7}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "RA_Integration.Data.rcheevos", "rcheevos_lib\RA_Integration.Data.rcheevos.vcxproj", "{9D55EBE7-1392-4FA1-A9B9-F022F764CE35}"
```

Comment:
Recommend just calling this `rcheevos.vcxproj`. It's not really part of RA_Integration until it's linked.

Reply:
I don't think it deserves its own subdirectory either. Just another vcxproj/filter in the src directory.

Reply:
It's in the rcheevos folder now (not part of submodule), the build goes to `solution_dir\bin(obj)\config\rcheevos` as well.

Reply:
It can't be in the `rcheevos` folder unless it's part of the submodule, or it won't be committed.

Reply:
So where should it go? RA_Integration folder?

Reply:
The project name can't be called rcheevos or else it'll go to the rcheevos folder by default.

Reply:
After you create the project, you should be able to remove the project reference from the solution, manually move the vcxproj and filters file into the src directory, then readd the project to the solution. At that point, you can start moving the files into the new project.

Reply:
Alright I'll try that. Been currently trying to rebase my master from RetroAchievements/master for future PRs. They will be identical again. I'm probably going to rebase master everytime I do a PR so it doesn't take this long. It's alright, it's my fault.

Reply:
Alright, this is done. My master branch matches RA_Achievements/master. Except for one commit "complete rebase", but that can't be helped.